includes/Rtree.h

```diff
@@ -0,0 +1,26 @@
+#ifndef RTREE_H
+#define RTREE_H
+
+#include "Node.h"
```

Comment:
It's almost always better to include using the absolute path (absolute in a sense relative to inclusion base path) like `#include <Node.h>`.  The `"path"` often surprises us.

Reply:
Done, thanks!